versions.tf

```diff
@@ -1,15 +1,19 @@
 terraform {
-  # We want to hold off on 0.13 until we have tested it.
+  # We want to hold off on 0.13 or higher until we have tested it.
   required_version = "~> 0.12.0"
 
   # If you use any other providers you should also pin them to the
   # major version currently being used.  This practice will help us
   # avoid unwelcome surprises.
   required_providers {
-    # 3.43.0 is the first version of the Terraform AWS provider that
-    # correctly allows the local_ipv4_network_cidr and
-    # remote_ipv4_network_cidr keys of the aws_vpn_connection resource
-    # to be non-/32 CIDRs.
+    # Version 3.38.0 of the Terraform AWS provider is the first
+    # version to support default tags.
+    # https://www.hashicorp.com/blog/default-tags-in-the-terraform-aws-provider
```

Comment:
Since we're no longer pinning on this version does this comment still make sense on its own?

Reply:
It's not on its own.  The comment about 3.43.0 follows.

Reply:
I think @mcdonnnj is saying that we no longer need the comment about `3.38.0`, since we are requiring `3.43.0` now.

Reply:
Yes, I meant is the comment redundant since we are pinning a later version in this project.

Reply:
See commit 5ef1301.